exercise/static/exercise/chapter.js

```diff
@@ -104,10 +104,7 @@
     },
 
     renderMath: function() {
-      if (typeof window.MathJax === "undefined") {
-        return;
-      }
-      window.MathJax.Hub.Queue(["Typeset", window.MathJax.Hub, this.modalElement.get(0)]);
+      $.fn.aplusTypesetMath(this.modalElement.get(0))
```

Comment:
```suggestion
      this.modalElement.aplusTypesetMath();
```
After you have fixed the plugin definition, I assume this line would be changed to this.